zarr/intro.qmd

```diff
@@ -7,36 +7,37 @@ subtitle: Chunked, Compressed N-Dimensional Arrays
 
 [Zarr](https://zarr.dev/), despite its name, is not a scary format. It's designed for data that is too big for users' machines, but Zarr makes data small and organizes it in a way where users can take just the bits they need or distribute the load of processing lots of those bits (stored as chunks) across many machines.
 
-The Zarr data format is a community-maintained format for large-scale n-dimensional data. A Zarr store consists of compressed and chunked n-dimensional arrays. Zarr's flexible indexing and compatibility with object storage lends itself to parallel processing.
+The Zarr data format is a community-maintained format for large-scale n-dimensional data. A Zarr store consists of chunked and compressed n-dimensional arrays. Zarr's flexible indexing and compatibility with object storage lends itself to parallel processing.
 
-A Zarr chunk is Zarr's unit of data storage. Each chunk of a Zarr array is an equally-sized block of the array within a larger Zarr store comprised of one or more arrays and array groups. These blocks or chunks of data are stored separately to make reading and updating small chunks more efficient.
+A Zarr chunk is Zarr's unit of data storage. Each chunk of a Zarr array is an equally-sized block of the array within a larger Zarr store. The larger Zarr store is comprised of one or more arrays and array groups. The Zarr chunks are normally stored in separate objects in object storage to make reading and updating individual chunks more efficient.
 
-Read more in the official tutorial: [Zarr Tutorial](https://zarr.readthedocs.io/en/stable/tutorial.html)
+Read more in the official zarr-python user guide: [Zarr User Guide](https://zarr.readthedocs.io/en/stable/user-guide/)
 
 ## Zarr Version 2 and Version 3
 
 ::: {.callout-important}
-Zarr Version 3 is underway but not released yet, so all the examples in this guide are for Zarr Version 2 data. The concepts in this page are consistent across both Zarr Version 2 and Zarr Version 3, however some metadata field names and organization are changing from Version 2 to version 3. 
+Zarr Version 3 represents a breaking change in the format but the concepts remain largely the same, however some metadata field names and organization have changed. Zarr Version 3 support is included in the canonical Python implementation -- zarr-python -- as of January 2025 (read more in the [release blog post](https://zarr.dev/blog/zarr-python-3-release/)). Zarr Versions 2 data is still usable in newer versions of the zarr-python library. All the examples in this guide are for Zarr Version 3 data and use zarr-python >= 3.
```

Comment:
Is "breaking change in the format" the way to describe it? I think of it more as just a new approach and specification for the same array-based data model